archive/archive_api/src/archive_api.py

```diff
@@ -38,7 +41,7 @@
 
 api.namespaces.clear()
 ns_ingests = api.namespace('ingests', description='Ingest requests')
-
+ns_assets = api.namespace('assets', description='Asset lookup requests')
 
```

Comment:
these are storage manifests rather than bags, so - dunno, up to you

Reply:
I took it to be the `/bags/XX-XXX` from here:
https://github.com/wellcometrust/platform/tree/master/docs/rfcs/002-archival_storage

Reply:
hmm. yes it seems to be. I find calling it a bag misleading, since it's not the bagit format, but rather a type which just happens to be called Bag that contains what the RFC calls a storage manifest. Is that distinction important? ( cc @kenoir )